pkg/operator/controllers/machine/machine_controller_test.go

```diff
@@ -24,167 +24,47 @@ import (
 )
 
 func TestMachineReconciler(t *testing.T) {
-	newFakeMao := func(diskSize string, imagePublisher string, vmSize string) *maofake.Clientset {
-		master0 := &machinev1beta1.Machine{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      "foo-hx8z7-master-0",
-				Namespace: machineSetsNamespace,
-				Labels:    map[string]string{"machine.openshift.io/cluster-api-machine-role": "master"},
-			},
-			Spec: machinev1beta1.MachineSpec{
-				ProviderSpec: machinev1beta1.ProviderSpec{
-					Value: &runtime.RawExtension{
-						Raw: []byte(`{
-"apiVersion": "azureproviderconfig.openshift.io/v1beta1",
-"kind": "AzureMachineProviderSpec",
-"osDisk": {
-"diskSizeGB": 512
-},
-"image": {
-"publisher": "azureopenshift",
-"offer": "aro4"
-},
-"vmSize": "Standard_D8s_v3"
-}`)},
-				},
-			},
-		}
+	// Fake cluster with AZs
+	newFakeMao1 := func(diskSize, imagePublisher, vmSize, masterDiskSize, masterVmSize string) *maofake.Clientset {
```

Comment:
The fourth parameter, `masterDiskSize`, is not actually used in any test cases (and based on my reading of [`pkg/operator/controllers/machine/machine.go`](https://github.com/Azure/ARO-RP/blob/38459f0449e434307eea630f5e28bf1db0bb96bf/pkg/operator/controllers/machine/machine.go#L59) it doesn't need to be). So I would remove it from this function.

Reply:
Thanks - I was playing with that on Friday and forgot to remove it.